setup.py

```diff
@@ -1,23 +1,40 @@
+import os
+import re
+import codecs
 from setuptools import setup, find_packages
-from codecs import open
-from os import path
 
-here = path.abspath(path.dirname(__file__))
+here = os.path.abspath(os.path.dirname(__file__))
+
+
+def read(*folders):
+    with codecs.open(os.path.join(here, *folders), encoding='utf-8') as f:
+        return f.read()
+
+
+def find_version(*file_paths):
+    version_file = read(*file_paths)
+    version_match = re.search(
+        r"^__version__ = ['\"]([^'\"]*)['\"]",
+        version_file,
+        re.M,
+    )
```

Comment:
Is necessary the last comma? for the way how you indent this code and this last comma, it seems to be a tuple.
Now in a very personal comment, I prefer this way:
```python
version_match = re.search(r"^__version__ = ['\"]([^'\"]*)['\"]",
                          version_file,
                          re.M)
```

Reply:
solved in commit b2f08aba5dff0c10a7c217c824e0086e5472c060